views/user/quiz/take-quiz.twig

```diff
@@ -139,10 +144,12 @@
     }
 </script>
 
+<script src="../lib/easytimer/dist/easytimer.min.js"></script>
```

Comment:
This line belongs in the header. See https://github.com/Deadpikle/pathfinder-bible-experience-prep-quiz-master/blob/3483d2d2395fa0aa11e3b4c75c6dfa8b7eaea1a1/views/admin/users/view-users.twig#L3-L7 for an example of how libraries should be referenced.